discovery-provider/src/api/v1/playlists.py

```diff
@@ -24,3 +26,18 @@ def get(self, playlist_id):
         playlists = list(map(extend_playlist, playlists))
         response = api_helpers.success_response(playlists, 200, False)
         return response
+
+playlist_tracks_response = make_response("playlist_tracks_response", ns, fields.List(fields.Nested(track)))
+
+@ns.route("/<string:playlist_id>/tracks")
+class PlaylistTracks(Resource):
```

Comment:
Let's just make sure this thing returns the proper 404 response if we pass in a playlist that doesn't exist. there should be a helper that makes it easier